src/Card/Card.js

```diff
@@ -1,12 +1,26 @@
+import cl from 'classnames';
 import './Card.scss';
 
-function Card({ className, caption, content }) {
+function Card({ className, caption, content,
+                checkbox = false, selected = false,
+                onSelect = undefined }) {
+
+  const onHeaderClick = () => {
+    return onSelect(!selected);
+  };
+
   return (
     <div className={className}>
 
-      <div className={'Card'}>
-        <div className={'Card__header'}>
-          {caption}
+      <div className={cl('Card', {'Card__selected': selected})}>
+        <div
+          className={cl('Card__header', {'Card__clickable': checkbox})}
+          onClick={checkbox ? onHeaderClick : undefined}
+        >
+          <div>{caption}</div>
+          {checkbox &&
+            <input type={'checkbox'} className={'Card__clickable'} checked={selected} readOnly />
+          }
```

Comment:
I think it is redundant.

Reply:
Ok, but how could user know if it is a selectable card then?

Reply:
Can you please provide me text which describes that checkbox may be absent?

Reply:
Sorry. Didn't get you, to be honest